### DISCUSSION TITLE
Value on chart data

### DISCUSSION THREAD

**Discussion by sofyanlabs** [Budibase/budibase#4802] [Help]
Hello I want to display the total income per day, the database I use is number.
Example : 
3 march - 8500
4 march - 6300

However, I want the data displayed on the chart to use comma, what can be done?

Example :
3 march - 8,500
4 march - 6,300

I think use with javascript or handlebar? but i can't see this feature on chart data

**Comment by mjashanks** [comment]
Hey @sofyanlabs - We don't have a way to do this at the moment. There's work to do on charts for sure, this is a good idea

**Comment by gerhardmpl** [comment]
I do not know the chart data components (yet), so my suggestion might be not working here. But it works with a `bbdb` table column defined as `formular` where you could use the _Intl.NumberFormat_ function (naughtily copied from the internet). Let's assume you have a table column `sales` which is of the type `numer` and another column `sales_formated` of the type `formular` using this Javascript code:

```
var formatter = new Intl.NumberFormat('en-US', {
  style: 'currency',
  currency: 'USD',
});

return formatter.format($("sales"));
```
If `sales` is _1500_ than `sales_formated` would be _$1,500.00_. There are more options to explore with _Intl.NumberFormat_. Maybe that can give you an idea on how to get going.
